packages/loader/driver-definitions/src/storage.ts

```diff
@@ -210,6 +210,7 @@ export enum LoaderCachingPolicy {
 
 export interface IDocumentServicePolicies {
     readonly caching?: LoaderCachingPolicy;
+    readonly noDeltaStream?: boolean;
```

Comment:
can we rename this to storageOnly as well to be consistent

Reply:
some comments about what this policy does would be good too

---
In reply to: [575506536](https://github.com/microsoft/FluidFramework/pull/5137#discussion_r575506536) [](ancestors = 575506536)